example/ios/Podfile

```diff
@@ -8,15 +8,15 @@ target 'BlobCourierExample' do
 
   use_react_native!(:path => config["reactNativePath"])
 
-#  pod 'react-native-blob-courier', :path => '../..'
```

Comment:
Missed that after running some testing some changes on another issue. Would very probably have been caught by #45, so don't discard this issue just yet.